.github/workflows/test.yml

```diff
@@ -0,0 +1,63 @@
+name: CI-testing
+
+on:
+  push:
+    branches: [ develop, master, truss ]
+  pull_request:
+    branches: [ develop, master, truss ]
```

Comment:
unless there is a particular reason for restricting the test runs to these branches, I would set the action to run on push and pull request on any branch, i.e.

```yaml
on:
  [push, pull_request]
jobs:
  ...
```

Reply:
Done. Thanks for the suggestion.